src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+$days: 31;
+$month-length: 28, 29, 30, 31;
+$start-days: (
+  'mon': 0,
+  'tue': 101px,
+  'wed': 202px,
+  'thu': 303px,
+  'fri': 404px,
+  'sat': 505px,
+  'sun': 606px,
+);
+
+html {
+  height: 100%;
+}
+
 body {
+  height: 100%;
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
+  padding: 0;
+  list-style: none;
+  cursor: pointer;
+
+  @each $start-day, $offset in $start-days {
+    &--start-day-#{$start-day} .calendar__day--1 {
+      margin-left: $offset;
+    }
+  }
+
+  @for $i from 1 through 3 {
+    &--month-length-#{31 - $i} :nth-last-child(-n + #{$i}) {
+      display: none;
+    }
+  }
+}
+
+.calendar__day {
+  position: relative;
+  display: flex;
+  color: black;
+  box-sizing: border-box;
+  width: 100px;
+  height: 100px;
+  border: 1px solid black;
+  background-color: #eee;
+  transition-duration: 0.2s, 0.5s;
```

Comment:
The `transition-duration` property is set with two values, but `transition-property` has only one property listed. If you intend to transition multiple properties, make sure to list them in `transition-property`, or use the shorthand `transition` to include all necessary values.